packages/samples/specs/resource-manager/resource-common-properties/managed-identity/main.tsp

```diff
@@ -0,0 +1,58 @@
+import "@typespec/http";
+import "@typespec/rest";
+import "@typespec/versioning";
+import "@azure-tools/typespec-azure-core";
+import "@azure-tools/typespec-azure-resource-manager";
+
+using TypeSpec.Http;
+using TypeSpec.Rest;
+using TypeSpec.Versioning;
+using Azure.ResourceManager;
+using Azure.Core;
+
+@armProviderNamespace("Contoso.ManagedIdentityExample")
+@service({
+  title: "Contoso.ManagedIdentityExample",
+})
+@OpenAPI.info({
+  version: "2023-03-01-preview",
```

Comment:
We should not follow this pattern in examples, we should have a Version enum with a single version and decorate the version with `useDependency` and `armCommonTypesVersion`